opm/parser/eclipse/CMakeLists.txt

```diff
@@ -271,7 +272,7 @@ EclipseState/Tables/RtempvdTable.hpp
 EclipseState/Tables/OilvisctTable.hpp
 EclipseState/Tables/GasvisctTable.hpp
 EclipseState/Tables/WatvisctTable.hpp
-EclipseState/Tables/PvtgOuterTable.hpp
+EclipseState/Tables/PvtgTable.hpp
```

Comment:
sorry, about the confusion: the patch is a bit twisted here, so it created a small mess in my brain (it seemed like PvtoTable.hpp was replaced by PvtxTable.hpp, but PvtoTable.hpp is re-added above)